Microsoft.Toolkit.Uwp.UI.Controls/Graph/ProfileCard/ProfileCard.xaml

```diff
@@ -0,0 +1,93 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls.Graph"
+    xmlns:converters="using:Microsoft.Toolkit.Uwp.UI.Converters">
+
+    <controls:DisplayModeConverter x:Key="DisplayModeConverter" />
+    <converters:StringVisibilityConverter x:Key="StringVisibilityConverter" />
+
+    <Style x:Key="ProfileCardImageBorderStyle"
+           TargetType="Border">
+        <Setter Property="Width" Value="80" />
+        <Setter Property="Height" Value="80" />
+    </Style>
+
+    <Style x:Key="ProfileCardImageEllipseStyle"
+           TargetType="Ellipse">
+        <Setter Property="Width" Value="64" />
+        <Setter Property="Height" Value="64" />
+        <Setter Property="VerticalAlignment" Value="Center" />
+    </Style>
+
+    <Style x:Key="ProfileCardLargeProfileStackPanelStyle"
+           TargetType="StackPanel">
+        <Setter Property="Height" Value="80" />
+        <Setter Property="Orientation" Value="Horizontal" />
+    </Style>
+
+    <Style x:Key="ProfileCardLargeProfileTextBlockStyle"
+           TargetType="TextBlock">
+        <Setter Property="FontSize" Value="24" />
+        <Setter Property="Margin" Value="0 0 0 4" />
+    </Style>
+
+    <Style TargetType="controls:ProfileCard">
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:ProfileCard">
+                    <Grid>
```

Comment:
This feels like each of these modes should be a different datatemplates for a ContentPresenter.  Then it'd be easy to add a custom template mode too to restyle the control easier.  Otherwise, we should use x:Load as we wouldn't want to load all the other modes in all cases.

Reply:
Done and committed. have replaced DisplayModeConverter with datatemplate in ProfileCard control.